FightPandemics.xcodeproj/project.pbxproj

```diff
@@ -532,6 +534,14 @@
 			path = OfferRequest;
 			sourceTree = "<group>";
 		};
+		C018DD762472EE0800EEB3FC /* TabBarUI */ = {
```

Comment:
let's organize this under `Features/TabBar` instead of `Features/Post/CreatePost`